### PR TITLE
[WIP]DO NOT MERGE: Draft to add command in place of KCS solution

### PR DESCRIPTION
**Pull request by opayne1**
DO NOT MERGE!! This is a draft to incorporate some feedback we have received about the current method of KCS solutions in z-stream release notes.  

Issue:
See [OSDOCS-3615](https://issues.redhat.com/browse/OSDOCS-3615) for more information

Link to docs preview (VPN Required):
http://file.rdu.redhat.com/opayne/z-stream-image-draft/release_notes/ocp-4-10-release-notes.html#ocp-4-10-z

### CODE REVIEW

**Review by thiagoalessio — APPROVED** [review]
/lgtm

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -2956,3 +2956,22 @@ link:https://access.redhat.com/solutions/6962052[{product-title} 4.10.17 contain
...
+
+Issued: yyyy-mm-dd
+
+{product-title} release 4.10.z is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:xxxx[RHBA-2022:xxxx] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.z --pullspecs
```
Comment (kalexand-rh) [inline release_notes/ocp-4-10-release-notes.adoc:2971]: I think this is a fine way to provide the data, and I'm glad that support suggested it. I have two requests, then it LGTM:
* Swap out (or explain) the`.z` part of this command 
* Confirm that you don't need to update `oc` before you run the command

**Comment by opayne1** [comment]
Putting this through the CM process to update all z-streams (4.6+) to replace the KCS link with `oc` command. I started on this pre-4.11 GA, but would like to implement as soon as possible as this was a request from a customer. This will be a dummy PR where the acks will live. This PR will not get merged.

Tagging the correct folks here for CM acks:

- @xltian 
- @sferich888 

cc: @kalexand-rh @Wiharris @JoeAldinger

**Comment by opayne1** [comment]
If you would like to see an example, I have implemented with the 4.11.0 GA release text: https://docs.openshift.com/container-platform/4.11/release_notes/ocp-4-11-release-notes.html#ocp-4-11-0-ga

**Comment by opayne1** [comment]
Closing this PR as all branches were merged.

**Pull request closed without merging**